yamllint/rules/comments.py

```diff
@@ -67,6 +70,7 @@
 ID = 'comments'
 TYPE = 'comment'
 CONF = {'require-starting-space': bool,
+        'ignore-shebang': bool,
```

Comment:
`ignore-shebangs` (with an **s** at the end) would be more consistent with other rules, in my opinion. What do you think?

Reply:
I agree. :)